python/nano/src/bigdl/nano/pytorch/amp/bfloat16.py

```diff
@@ -46,6 +48,40 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         return super().__exit__(exc_type, exc_val, exc_tb)
 
 
+class RedirectStream(object):
```

Comment:
where is this used?

Reply:
The contextmanager was used to capture the output of dnnl verbose output , because python `contextlib.redirect_stdout` only works for python package. I removed the BF16 check related method in `PytorchIPEXJITBF16Model` because it is not necessary for IPEX BF16. But this contextmanager is still useful for [`bigdl.nano.pytorch.amp.bfloat16.BF16Model`](https://github.com/intel-analytics/BigDL/blob/main/python/nano/src/bigdl/nano/pytorch/amp/bfloat16.py#L115) to verify if bf16 instructions are used, so I keep it in this pr.